src/select/select.js

```diff
@@ -350,6 +359,40 @@ class Select extends React.Component<PropsT, StatelessStateT> {
     const {type, multiple} = this.props;
     return type === TYPE.search ? true : multiple;
   }
+
+  handledHotKeys(
+    // $FlowFixMe
+    e: SyntheticEvent<EventTarget> | KeyboardEvent,
+    option?: ?OptionT,
+  ) {
+    switch (e.key) {
+      case KEY_STRINGS.ArrowDown:
+      case KEY_STRINGS.Space:
+        if (e.key === KEY_STRINGS.Space && this.props.type === TYPE.search) {
+          return;
+        }
+        this.setState({isDropDownOpen: true});
+        e.preventDefault();
+        e.stopPropagation();
+        return true;
+      case KEY_STRINGS.Escape:
+        this.setState({isDropDownOpen: false});
+        return true;
+      case KEY_STRINGS.Enter:
+        if (option) {
+          this.onChange(e, STATE_CHANGE_TYPE.select, option.id, option.label);
+        }
+        return;
+      case KEY_STRINGS.Backspace:
+        if (this.isMultiple() && !this.state.textValue) {
+          const selectedOptions = this.state.selectedOptions.slice();
+          selectedOptions.pop();
+          this.setState({selectedOptions});
+          return true;
+        }
```

Comment:
It looks like it still deletes the previous tag when I delete the last character in the input:

![kapture 2018-09-19 at 10 29 06](https://user-images.githubusercontent.com/875591/45770367-092c1400-bbf7-11e8-9494-a72e6acad072.gif)

Can we make it so that it only deletes when the input is empty?